charts/kube-plex/values.yaml

```diff
@@ -63,8 +95,29 @@ rbac:
 nodeSelector:
   beta.kubernetes.io/arch: amd64
 
+configMap:
+  # Enable init script that will read all environment variables starting with PLEX_PREFERENCE_
+  # and take the value as the Key:Value option to set in Plex Preference.xml
+  # You can use extraEnv to add the addtional ENV's to the container.
+  # --set extraEnv.PLEX_PREFERENCE_1="FriendlyName=plex-kube-plex-test1" `
+  # --set extraEnv.PLEX_PREFERENCE_2="EnableIPv6=0" `
+  # --set extraEnv.PLEX_PREFERENCE_3="logDebug=0" `
+  # --set extraEnv.PLEX_PREFERENCE_4="DisableTLSv1_0=1" `
+  # --set extraEnv.PLEX_PREFERENCE_5="LanNetworksBandwidth=xxx.xxx.xxx.0/18\,xxx.xxx.xxx.0/24\,xxx.xxx.xxx.0/24" `
+  # --set extraEnv.PLEX_PREFERENCE_6="TranscoderQuality=2" `
+  # --set extraEnv.PLEX_PREFERENCE_7="TreatWanIpAsLocal=0" `
+  # --set extraEnv.PLEX_PREFERENCE_8="TranscoderH264BackgroundPreset=fast" 
+  plexPreferences:
+    enabled: false
+    name: 41-plex-preferences
+    defaultMode: 493
+    mountPath: /etc/cont-init.d/41-plex-preferences
+    subPath: 41-plex-preferences
```

Comment:
I'm not quite understanding why options to change the name, mountpath, subpath or mode of this configmap are exposed here, nor why `enabled` is exposed either.

I'd expect the init script to not take any action if these `PLEX_PREFERENCE` vars are not set (i.e. the default), and I don't see anywhere/any reason why a user would want to provide their own ConfigMap to use instead of this one.

If we want to allow a user to inject arbitrary shell code that is executed before PMS starts, I think a dedicated variable that indicates what it is (i.e. `preStartShellScript`) would be clearer.

As an alternative way of structuring this `values.yaml`, what about if there was:

```
# An optional list of Plex Preferences.xml keys/values that will be set/updated before the Plex Media Server process starts.
plexPreferences:
  FriendlyName: plex-kube-plex-test1
  EnableIPv6: "0"
  logDebug: "0"
```

etc?